tests/io/test.directory.js

```diff
@@ -61,85 +55,56 @@ describe('Directory.getFiles()', () => {
+  it('should reject if not a file that exists', async () => {
     const myDirectory = new Directory('tests/fixtures/io/');
-    return myDirectory.getFiles()
-      .then(() => {
-        return myDirectory.getPath('whatever')
-          .then(unexpectedSuccess)
-          .catch((err) => {
-            expect(err.message).toContain(
-              '"whatever" does not exist in this dir.'
-            );
-          });
-      });
+
+    await myDirectory.getFiles();
+    await expect(
+      myDirectory.getPath('whatever')
+    ).rejects.toThrow('"whatever" does not exist in this dir.');
   });
 
-  it('should reject if path does not start with base', () => {
+  it('should reject if path does not start with base', async () => {
     const myDirectory = new Directory('tests/fixtures/io/');
     myDirectory.files = {
       '../file1.txt': {},
     };
-    return myDirectory.getPath('../file1.txt')
-      .then(unexpectedSuccess)
-      .catch((err) => {
-        expect(err.message).toContain('Path argument must be relative');
-      });
+
+    await expect(
+      myDirectory.getPath('../file1.txt')
+    ).rejects.toThrow('Path argument must be relative');
   });
 
-  it("should reject if path starts with '/'", () => {
+  it("should reject if path starts with '/'", async () => {
     const myDirectory = new Directory('tests/fixtures/io/');
     myDirectory.files = {
       '/file1.txt': {},
     };
-    return myDirectory.getPath('/file1.txt')
-      .then(unexpectedSuccess)
-      .catch((err) => {
-        expect(err.message).toContain('Path argument must be relative');
-      });
+
+    await expect(
+      myDirectory.getPath('/file1.txt')
+    ).rejects.toThrow('Path argument must be relative');
   });
 });
 
 describe('Directory.getFileAsStream()', () => {
-  it('should return a stream', () => {
+  it('should return a stream', async () => {
     const myDirectory = new Directory('tests/fixtures/io/');
-    return myDirectory.getFiles()
-      .then(() => {
-        return myDirectory.getFileAsStream('dir2/dir3/file3.txt');
-      })
-      .then((readStream) => {
-        return new Promise((resolve, reject) => {
-          let content = '';
-          readStream
-            .on('readable', () => {
-              let chunk;
-              // eslint-disable-next-line no-cond-assign
-              while ((chunk = readStream.read()) !== null) {
-                content += chunk.toString();
-              }
-            })
-            .on('end', () => {
-              resolve(content);
-            })
-            .on('error', (err) => {
-              reject(err);
-            });
-        })
-          .then((content) => {
-            expect(content).toEqual('123\n');
-          });
-      });
+    await myDirectory.getFiles();
+
+    const readStream = await myDirectory.getFileAsStream('dir2/dir3/file3.txt');
+
+    const content = await readStringFromStream(readStream);
+    expect(content).toEqual('123\n');
```

Comment:
@rpl thanks for the very helpful Jest link. Now I have a couple of suggestions about using Jest methods. For example, maybe here we can replace lines 106, 107 with 
await expect(readStringFromStream(readStream).resolves.toBe('123\n');

Reply:
@hameleonka 👍 sure! go for it!

Reply:
@rpl I've changed some Jest methods in this file